samples/azure-quantum/chemistry/Hamiltonian/Approximate Energy using Jordan-Wigner transformation.ipynb

```diff
@@ -0,0 +1,813 @@
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.qpu...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 7\n",
+      "   Job ID: d0f1c1b4-62e8-48a4-bd85-5df0fa3485c8\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.qpu...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 8\n",
+      "   Job ID: 54313164-412f-46b8-bf37-3a760226e2c7\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.qpu...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 9\n",
+      "   Job ID: 0ea376fd-8e72-478c-bfbe-c4f5c022e040\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.qpu...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 12\n",
+      "   Job ID: 0eecd3a6-ed34-4065-be6a-d9f848fcbbe6\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.qpu...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 13\n",
+      "   Job ID: 0425662e-85ad-4ba5-b9d3-12c16ffa5ea1\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.qpu...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 16\n",
+      "   Job ID: aa93563a-6def-4684-b217-353593886598\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.qpu...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 17\n",
+      "   Job ID: 71b894a1-e252-40ae-9274-ffb1063163ae\n"
+     ]
+    }
+   ],
+   "source": [
+    "jobs_hw = [\n",
+    "    qsharp.azure.submit(GetHamiltonianTermH2, nOp=n_op, shots=1000, jobName=f\"Hamiltonian term {n_op}\") \n",
+    "    for n_op, coeff in enumerate(coeffs) if coeff != 0\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f65fb519",
+   "metadata": {},
+   "source": [
+    "Check if the jobs succeeded."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "id": "927be51a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded',\n",
+       " 'Succeeded']"
+      ]
+     },
+     "execution_count": 59,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "[qsharp.azure.status(j.id).status for j in jobs_hw]"
```

Comment:
For this cell, I added some more fields to make it a bit more visually interesting. In the demo I had

```
jobs_status = [qsharp.azure.status(j) for j in jobs_hw_past]
[j.status + " | " + j.name + " | " + j.target + " | " + j.creation_time + " | " + j.end_execution_time 
 for j in jobs_status]
```